figures/scripts/fig2_figS3-oncoprint-landscape.R

```diff
@@ -369,6 +426,14 @@ for (type_iter in seq_along(data_input_list)) {
     create_legend(legend_df$cancer_group_display,
                   legend_df$cancer_group_hex,
                   legend_output_pdf)
+    
+    # Export CSV for Zenodo upload, after filtering to histology_goi genes
+    maf_export <- histology_maf_object_df$maf_df %>%
```

Comment:
suggested naming above
```suggestion
    maf_export <- histology_maf_pair$maf_df %>%
```